### DISCUSSION TITLE
Cache usage

### DISCUSSION THREAD

**Discussion by dippi9845** [laravel/framework#44856] [Q&A]
There is a way to check if a cache is occuping all the memory of the database used ?
handle that case, and maybe free some keys

**Comment by henzeb** [comment]
About what cache database are we talking?

**Reply by dippi9845** [reply]
I'm referring to this [cache class](https://laravel.com/docs/9.x/cache) of laravel. I mean there a way to abstract from the database implementation, and a the same time knowing the cache space used, or alternatively know how much is remaining from the saturation of the database cache.

**Reply by henzeb** [reply]
Yes, but that has no option for that. Some databases might.

**Reply by dippi9845** [reply]
I still haven't used the cache, so could you tell me which database can handle it? and where I can suggest this feature, (I mean a method that abstract from the database implementation) ?
